routes/index.js

```diff
@@ -75,7 +79,7 @@ module.exports = function(app) {
     });
 
     // This must come before /callbacks/:uuid to be matched explictly.
-    app.post('/callbacks/ctc-agent-answers', (request, response) => {
+    app.post('/callbacks/ctc-agent-answers', twilio.webhook({validate: shouldValidate}), (request, response) => {
```

Comment:
is this like middleware?

Reply:
Yes, it's middleware provided by [twilio-node](https://github.com/twilio/twilio-node). You can read about it in [lib/webhooks/webhooks.js](https://github.com/twilio/twilio-node/blob/master/lib/webhooks/webhooks.js#L92).